MoviesMVP/MoviesMVP/Services/KeychainService.swift

```diff
@@ -0,0 +1,39 @@
+// KeychainService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import KeychainAccess
+
+final class KeychainService {
+    // MARK: - Private Enum
+
+    private enum Constants {
+        static let apiValueString = "a5b0bb6ebe58602d88ccf2463076122b"
+        static let apiKeyString = "apiKey"
+        static let keychainServiceString = "ru.MoviesMVP"
+    }
+
+    // MARK: - Public property
+
+    static let shared = KeychainService()
+
+    // MARK: - Initializer
+
+    private init() {}
+
+    // MARK: - Public methods
+
+    func getAPIKey() -> String {
```

Comment:
```suggestion
    func getAPIKey() -> String {
```
Глянь как этот сервис Даниил реализовал